config/routes.rb

```diff
@@ -115,11 +115,8 @@
 
     resources :training_sessions do
       collection do
```

Comment:
This `collection` method is supposed to be used to add methods to a collection of resources (like training_sessions). Usually that would be something like search.

Instead, you should use `member` (see [rails routing guide](http://guides.rubyonrails.org/routing.html#adding-more-restful-actions)). This will mean that the url will include the id in like `/staff/training_sessions/IDHERE/certifiy_trainees` instead of the way it is now where it is `/staff/training_sessions/certifiy_trainees`